File: statistic-service/src/main/java/at/technikumwien/MainApp.java
package at.technikumwien;

import at.technikumwien.controller.HelloWorldController;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.boot.CommandLineRunner;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.context.annotation.ComponentScan;

@SpringBootApplication
public class MainApp implements CommandLineRunner {
    @Autowired
    private HelloWorldController hwController;

    public static void main(String[] args) {
        SpringApplication.run(MainApp.class, args);
    }

    @Override
    public void run(String... args) {
        hwController.sayHello();
    }
}

File: statistic-service/src/main/kotlin/at/technikumwien/MainApp.kt
package at.technikumwien

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.cloud.stream.annotation.EnableBinding
import org.springframework.cloud.stream.messaging.Sink

@SpringBootApplication
@EnableBinding(Sink::class)
class StatisticServiceApplication

fun main(args: Array<String>) {
    runApplication<StatisticServiceApplication>(*args)
}
